GLOSSARY.md

```diff
@@ -0,0 +1,21 @@
+# Glossary
+
+- **Desired state**
+
+    The aggregate of all configuration data for a system form its "Desired State". The "Desired State" of a system is defined as data sufficient to recreate the system from nothing so that instances of the system are behaviorally indinstinguishable.
+
+- **State Store**
+    
+    A system for storing versioned, immutable Desired States that provides access control and auditing on the chnages to the Desired State. Git may be configured as a State Store, but special precautions must be taken.
```

Comment:
```suggestion
    A system for storing versioned, immutable Desired States that provides access control and auditing on the changes to the Desired State. Git may be configured as a State Store, but special precautions must be taken.
```

Maybe link to what "Git may be configured as a State Store, but special precautions must be taken." means exactly?